cpp/include/cudf/detail/copy_if_else.cuh

```diff
@@ -177,7 +178,7 @@ std::unique_ptr<column> copy_if_else(
     make_fixed_width_column(data_type(type_to_id<Element>()),
                             size,
                             nullable ? mask_state::UNINITIALIZED : mask_state::UNALLOCATED,
-                            stream,
+                            stream.value(),
```

Comment:
This is a gradual transition. If I change every stream I find as I find them it's a huge rabbit hole. I need to break it up or it will be a massive unmanageable PR. I will be going back and searching and replacing stream.value() with stream as I change more functions.  Column factories are in part 2.